### PR TITLE
Documentation/upgrades: upgrade 3.0 doc

### PR DESCRIPTION
**Pull request by gyuho**
Tested and confirmed that it worked.

/cc @xiang90 @heyitsanthony

### CODE REVIEW

**Comment by gyuho** [comment]
Btw, tested 3.0-beta to 3.0 as well. Works well.

**Comment by heyitsanthony** [comment]
lgtm

File: Documentation/v2/upgrade_3_0.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+```
+
+```
+$ ETCDCTL_API=3 etcdctl endpoint health
+127.0.0.1:12379 is healthy: successfully committed proposal: took = 18.440155ms
+127.0.0.1:32379 is healthy: successfully committed proposal: took = 13.651368ms
+127.0.0.1:22379 is healthy: successfully committed proposal: took = 18.513301ms
+```
+
+[etcd-contact]: https://coreos.com/etcd/?
```
Comment (xiang90) [inline Documentation/v2/upgrade_3_0.md]: https://groups.google.com/forum/#!forum/etcd-dev

**Comment by xiang90** [comment]
move this to /doc/upgrades. also add a link here: https://github.com/coreos/etcd/blob/master/Documentation/docs.md#upgrading-and-compatibility

**Comment by gyuho** [comment]
@xiang90 Addressed. PTAL.

File: Documentation/docs.md [hunk 2]
```diff
@@ -45,6 +45,7 @@ To learn more about the concepts and internals behind etcd, read the following p
 ## Upgrading and compatibility
 
  - [Migrate applications from using API v2 to API v3][v2_migration]
+ - [Upgrading v2.3 to v3][v3_upgrade]
```
Comment (xiang90) [inline Documentation/docs.md]: updating v2.3 to v3.0

**Comment by xiang90** [comment]
lgtm

**Comment by gyuho** [comment]
thanks.

**Pull request merged**